src/feature-test/user_see_data_pending.test.js

```diff
@@ -0,0 +1,40 @@
+import '../setupTest.js'
+import { jsxEmptyExpression } from '@babel/types';
+import { SSL_OP_EPHEMERAL_RSA } from 'constants';
```

Comment:
What is this? A dependency? What does it do?

Reply:
Good catch. No idea where this came from.